bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/SomfyTahomaHandlerFactory.java

```diff
@@ -31,7 +30,40 @@
 import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
 import org.eclipse.smarthome.io.net.http.HttpClientFactory;
 import org.openhab.binding.somfytahoma.internal.discovery.SomfyTahomaItemDiscoveryService;
-import org.openhab.binding.somfytahoma.internal.handler.*;
```

Comment:
I don't get it, why is it that you like static star imports but hate regular star imports?

Reply:
It's just the autosave actions. I didn't spend any time on changing imports.